meta-lmp-base/recipes-core/images/initramfs-ostree-lmp-recovery.bb

```diff
@@ -9,10 +9,10 @@ SRC_URI = "\
 "
 
 PACKAGE_INSTALL = " \
-	busybox \
```

Comment:
Well, I'm unhappy with this. Recovery should be compact, so `busybox` is a good choice, whereas, for an ostree lmp image, we could choose a more functional package for base utils.